<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_run_popup">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="compiler_groupbox">
     <property name="title">
      <string>Compiler options</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="0" column="2">
       <widget class="QLabel" name="label_compiler">
        <property name="text">
         <string>Compiler</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="combobox_compiler">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_flags">
        <property name="toolTip">
         <string>Select &quot;Custom&quot; build type to specify compiler flags</string>
        </property>
        <property name="text">
         <string>Compiler flags</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineedit_compiler_flags">
        <property name="toolTip">
         <string>Select &quot;Custom&quot; build type to specify compiler flags</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_buildtype">
        <property name="text">
         <string>Build type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="combobox_buildtype"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="checkbox_groupbox">
     <property name="title">
      <string>Build options</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="checkbox_dmp">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Distributed memory parallel (DMP)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="combobox_server">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_server">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Interactive support</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkbox_smp">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Shared memory parallel (SMP)</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="checkbox_parallel">
        <property name="text">
         <string>Build solver in parallel (recommended for faster builds)</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkbox_verbose">
        <property name="text">
         <string>Verbose</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_command">
     <property name="text">
      <string>Build command</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineedit_command">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_progress">
     <property name="text">
      <string>Press &quot;Build solver&quot; to compile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressbar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushbutton_show_out">
        <property name="text">
         <string>Show build output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushbutton_clean">
        <property name="text">
         <string>Clean</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushbutton_build">
        <property name="text">
         <string>Build solver</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushbutton_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textbrowser">
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_error_buttons">
     <item>
      <widget class="QPushButton" name="button_first_error">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>First error</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_prev_error">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Previous error</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_next_error">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Next error</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combobox_compiler</tabstop>
  <tabstop>combobox_buildtype</tabstop>
  <tabstop>lineedit_compiler_flags</tabstop>
  <tabstop>checkbox_smp</tabstop>
  <tabstop>checkbox_dmp</tabstop>
  <tabstop>checkbox_parallel</tabstop>
  <tabstop>checkbox_verbose</tabstop>
  <tabstop>combobox_server</tabstop>
  <tabstop>lineedit_command</tabstop>
  <tabstop>pushbutton_show_out</tabstop>
  <tabstop>pushbutton_clean</tabstop>
  <tabstop>pushbutton_build</tabstop>
  <tabstop>pushbutton_cancel</tabstop>
  <tabstop>textbrowser</tabstop>
  <tabstop>button_first_error</tabstop>
  <tabstop>button_prev_error</tabstop>
  <tabstop>button_next_error</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
